src/js/ripple/serializedobject.js

```diff
@@ -209,8 +216,8 @@ SerializedObject.jsonify_structure = function(structure, field_name) {
 
       if (typeof structure.to_json === 'function') {
         output = structure.to_json();
-      } else if (structure instanceof BigInteger) {
-        output = ('0000000000000000' + structure.toString(16).toUpperCase()).slice(-16);
+      } else if (structure instanceof sjcl.bn) {
+        output = ('0000000000000000' + normalize_sjcl_bn_hex(structure.toString()).toUpperCase()).slice(-16);
```

Comment:
The 16 leading zeroes is clearly for ensuring Uint64 types pad correctly.  With .slice(-16) you couldn't get any larger either.  It would be nice to tighten this up at some point, but as you said, in another patch.